ignite/services/chain/build.go

```diff
@@ -29,6 +29,7 @@ const (
 	releaseChecksumKey           = "release_checksum"
 	modChecksumKey               = "go_mod_checksum"
 	buildDirchangeCacheNamespace = "build.dirchange"
+	consumerDevel                = "consumer_devel"
```

Comment:
what is this flag?

Reply:
cc @tbruyelle

Reply:
See a few lines below, it's a build tag that is appended to the other build tags, when the chain is a consumer chain. When set (which should be done only for non-production build), it doesn't append the `consumerante.MsgFilterDecorator` to the ante decorators. This makes it possible to send message to a local consumer chain even if the IBC handshake with the provider chain is not established.

I dug up this pattern from the Neutron chain, see https://github.com/neutron-org/neutron/blob/08c5a4406cedc62ef76b5c3d5cb7e45b91c594ae/app/ante_handler.go#L100-L106.